src/twisted/newsfragments/12169.bugfix

```diff
@@ -0,0 +1 @@
+Added more tests for ``twisted.internet.defer.inlineCallbacks`` decorator.
```

Comment:
What is the bug fix here ?

What was not working before in the public API?

-------

It looks like this should be a 12168.misc file.

Reply:
Fixed.